src/config/wmodules-github.c

```diff
@@ -9,15 +9,154 @@
+                merror("Invalid content for tag '%s' at module '%s'.", XML_INTERVAL, WM_GITHUB_CONTEXT.name);
+                return OS_INVALID;
+            }
+        } else if (!strcmp(nodes[i]->element, XML_TIME_DELAY)) {
+            github_config->time_delay = time_convert(nodes[i]->content);
+            if (github_config->time_delay == OS_INVALID){
+                merror("Invalid content for tag '%s' at module '%s'.", XML_TIME_DELAY, WM_GITHUB_CONTEXT.name);
+                return OS_INVALID;
+            }
+        } else if (!strcmp(nodes[i]->element, XML_ONLY_FUTURE_EVENTS)) {
+            if (!strcmp(nodes[i]->content, "yes"))
+                github_config->only_future_events = 1;
+            else if (!strcmp(nodes[i]->content, "no"))
+                github_config->only_future_events = 0;
+            else {
+                merror("Invalid content for tag '%s' at module '%s'.", XML_ONLY_FUTURE_EVENTS, WM_GITHUB_CONTEXT.name);
+                return OS_INVALID;
+            }
+        } else if (!strcmp(nodes[i]->element, XML_API_AUTH)) {
+            if (!(children = OS_GetElementsbyNode(xml, nodes[i]))) {
+                continue;
+            }
+            for (j = 0; children[j]; j++){
+                if (!strcmp(children[j]->element, XML_ORG_NAME)) {
+                    if (strlen(children[j]->content) == 0) {
+                        merror("Empty content for tag '%s' at module '%s'.", XML_ORG_NAME, WM_GITHUB_CONTEXT.name);
+                        OS_ClearNode(children);
+                        return OS_INVALID;
+                    }
+                    os_strdup(children[j]->content, github_config->org_name);
+                } else if (!strcmp(children[j]->element, XML_API_TOKEN)) {
+                    if (strlen(children[j]->content) == 0) {
+                        merror("Empty content for tag '%s' at module '%s'.", XML_API_TOKEN, WM_GITHUB_CONTEXT.name);
+                        OS_ClearNode(children);
+                        return OS_INVALID;
+                    }
+                    os_strdup(children[j]->content, github_config->api_token);
+                }
+            }
+            OS_ClearNode(children);
+        } else if (!strcmp(nodes[i]->element, XML_API_PARAMETERS)) {
+            if (!(children = OS_GetElementsbyNode(xml, nodes[i]))) {
+                continue;
+            }
+            for (j = 0; children[j]; j++){
+                if (!strcmp(children[j]->element, XML_EVENT_TYPE)) {
+                    if (strcmp(children[j]->content, "all") && strcmp(children[j]->content, "git") && strcmp(children[j]->content, "web")) {
+                        merror("Invalid content for tag '%s' at module '%s'.", XML_EVENT_TYPE, WM_GITHUB_CONTEXT.name);
+                        OS_ClearNode(children);
+                        return OS_INVALID;
+                    }
+                    os_strdup(children[j]->content, github_config->event_type);
```

Comment:
Check if `github_config->event_type` is not NULL. If not NULL, log a warning message indicating that the tag is repeated in the same block.